framework/spree/types/index.ts

```diff
@@ -0,0 +1,164 @@
+import type { fetchResponseKey } from '@framework/utils/create-customized-fetch-fetcher'
```

Comment:
We should always do relative paths inside the providers folder, otherwise build will fail for other providers